src/together/types/finetune.py

```diff
@@ -121,6 +146,7 @@ class FinetuneRequest(BaseModel):
     suffix: str | None = None
     # weights & biases api key
     wandb_key: str | None = None
+    training_type: FullTrainingType | LoRATrainingType | None = None
```

Comment:
Best to change it to just `TrainingType` perhaps? Or something like https://docs.python.org/3/library/typing.html#newtype to ensure that only subclasses of TrainingType can be used here

Reply:
I'm not sure `pydantic` will serialize NewType or TrainingType to right typings